endpoints.yml

```
@@ -0,0 +1,2100 @@
+---
+endpoints:
+  core:home:
+    - /core/
+  core:about:
+    - /core/about/
+  core:search:
+    - /core/search/
+  core:graphql:
+    - /core/graphql/
+  core:media_failure:
+    - /core/media-failure/
+  core:db_file_storage.download_file:
+    - /core/files/download/
+  core:worker_status:
+    - /core/worker-status/
+  core:render_jinja_template:
+    - /core/render-jinja-template/
+  core:theme_preview:
+    - /core/theme-preview/
+  core:metrics:
+    - /core/metrics/
+  circuits:circuit_terminations_swap:
+    - /circuits/circuits/<uuid:pk>/terminations/swap/
+  circuits:circuittermination_add:
+    - /circuits/circuits/<uuid:circuit>/terminations/add/
+  circuits:provider_list:
+    - /circuits/providers/$
+  circuits:provider_add:
+    - /circuits/providers/add/$
+  circuits:provider_import:
+    - /circuits/providers/import/$
+  circuits:provider_bulk_edit:
+    - /circuits/providers/edit/$
+  circuits:provider_bulk_delete:
+    - /circuits/providers/delete/$
+  circuits:provider:
+    - /circuits/providers/(?P<pk>[/.]+)/$
```

Comparing to the original file, these URLs should be directly usable (no placeholders), and based on the data (PKs, etc.) actually in the database at the time the file is generated, e.g.:

 ```suggestion
     - /circuits/providers/03f78bc9-2511-436d-a55f-e1954f8f5ebf/
```

I'd propose the following additional requirements based on the original file:
- Any URL pattern with a `<pk>` should result in two output URLs (two different object instances)
- Any "list" URL should include valid query parameters based on the amount of data present (e.g. `?per_page=10&page=5` would be a possibility if there are at least 50 records of the given type)
- The selection of values to the previous two points must be deterministic (e.g. random but with a reusable seed, similar to how our factories operate) so that rerunning the command on the same data set produces the same file
   - Ideally, changes to the data set should have narrow impact to the generated file, e.g. adding another Device record may change the `dcim:devices` URLs selected, but the `circuits:circuit` URL should not change.